### DISCUSSION TITLE
Date Picker label format

### DISCUSSION THREAD

**Discussion by jpfieber** [mProjectsCode/obsidian-meta-bind-plugin#352] [Q&A] [answered]
I'm using this to modify a frontmatter property that holds a date:
````
```meta-bind
INPUT[datePicker:dashDate]
```
````
Works great, but I'd like to modify the button so it shows the date in a different format than is used in the frontmatter.  Is it possible to have the datePicker button show that date as "cccc, MMMM d, yyyy" in the button after it's chosen, while the property remains in "yyyy-MM-dd" for easy use in Dataviewjs functions on the page?

**Comment by mProjectsCode** [comment, marked as answer]
No. The display and frontmatter format are the same.
